glide.lock

```diff
@@ -40,58 +34,49 @@ imports:
   - reporters
   - reporters/stenographer
   - types
-  - ginkgo/convert
-  - ginkgo/interrupthandler
-  - ginkgo/nodot
-  - ginkgo/testrunner
-  - ginkgo/testsuite
-  - ginkgo/watch
+  - internal/containernode
   - internal/leafnodes
+  - internal/spec
+  - internal/specrunner
 - name: github.com/onsi/gomega
-  version: 9574b2d94c03faf63bf1b296e21d9db950006ea4
+  version: 26ad09bd764b8f3b6a2489753642774559fe049c
   subpackages:
   - internal/assertion
   - internal/asyncassertion
   - internal/testingtsupport
   - matchers
   - types
   - internal/oraclematcher
+  - format
   - matchers/support/goraph/bipartitegraph
   - matchers/support/goraph/edge
   - matchers/support/goraph/node
   - matchers/support/goraph/util
-- name: github.com/opencontrol/doc-template
-  version: 88997261e3057776e761bb1460d6a53ae240a782
-  subpackages:
-  - docx
 - name: github.com/pmezard/go-difflib
-  version: d8ed2627bdf02c080bf22230dbb337003b7aba2d
+  version: ""
   subpackages:
   - difflib
 - name: github.com/stretchr/objx
-  version: cbeaeb16a013161a98496fad62933b1d21786672
+  version: ""
 - name: github.com/stretchr/testify
-  version: d77da356e56a7428ad25149ca77381849a6a5232
+  version: 26ad09bd764b8f3b6a2489753642774559fe049c
   subpackages:
   - mock
   - assert
-  - http
 - name: github.com/tg/gosortmap
-  version: 9e9c6c22cdeb75203f2b1ee5f7eb8d5d7832e7b0
-- name: github.com/urfave/cli
-  version: 3a5216227e14699bf7810b2573db60bf4b3f71b5
+  version: 26ad09bd764b8f3b6a2489753642774559fe049c
 - name: github.com/vektra/errors
-  version: c64d83aba85aa4392895aadeefabbd24e89f3580
+  version: 26ad09bd764b8f3b6a2489753642774559fe049c
 - name: golang.org/x/sys
-  version: b518c298ac9dc94b6ac0757394f50d10c5dfa25a
+  version: 26ad09bd764b8f3b6a2489753642774559fe049c
   subpackages:
   - unix
 - name: gopkg.in/fatih/set.v0
-  version: 27c40922c40b43fe04554d8223a402af3ea333f3
+  version: 26ad09bd764b8f3b6a2489753642774559fe049c
 - name: gopkg.in/yaml.v2
-  version: a83829b6f1293c91addabc89d0571c246397bbf4
+  version: 26ad09bd764b8f3b6a2489753642774559fe049c
 - name: vbom.ml/util
-  version: db5cfe13f5cc80a4990d98e2e1b0707a4d1a5394
+  version: 26ad09bd764b8f3b6a2489753642774559fe049c
```

Comment:
Looks like `glide rm github.com/opencontrol/doc-template` corrected the `version`s to reflect what's in `vendor/`, rather than what's the latest in the respective repositories. My bad from earlier!